packages/frontend/src/lib/ui/ShowNote.svelte

```diff
@@ -3,7 +3,8 @@
 </script>
 
 <script lang="ts">
-	import { saveAs } from 'file-saver'
+	import pkg from 'file-saver'
```

Comment:
It would be nice but as far as I know, there is no way to make it one line as long as `file-saver` is CJS. It seems like that package hasn't been updated for 3 years now.

Reply:
I mean like removing the dependency, and just copy pasta the actual library into the code base. It's 1 file. But i'll do it in another PR :)